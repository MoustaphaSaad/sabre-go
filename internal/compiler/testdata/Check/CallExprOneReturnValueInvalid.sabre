package main

func foo() float32 {
    return 1.0
}

func bar() int {
    return foo()
}
