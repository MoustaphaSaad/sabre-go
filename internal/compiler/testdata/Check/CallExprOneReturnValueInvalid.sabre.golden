>> 	    return foo()
>> 	           ^^^^^ 
Error[internal/compiler/testdata/Check/CallExprOneReturnValueInvalid.sabre:8:12]: incorrect return type 'float32', expected 'int'
